### ISSUE TITLE
WIPvsSOR3+FinalFight(arcade)

### ISSUE THREAD

**Issue by ArcadeHustle** [ArcadeHustle/CSI-Paprium#26]
![FDpBGM5WUAsb0Jj](https://user-images.githubusercontent.com/41961787/140684223-9643d724-1cae-40a1-855f-da5cc94be53d.jpeg)

**Comment by 2blackbar** [comment]
Yup, this is normal so why playin tha iNteRnEt PoLiCemAn ? Maybe educate yourself how animations are made and why people like to use the same keyframes and poses.

**Comment by ArcadeHustle** [comment]
Yet here you are @2blackbar trying to internet police us... #SeemsLegit 

"Educate yourself on why someone would be lazy and steal someone else's game models" - You sounding like a clown.
